src/Mvc/Mvc.ApiExplorer/src/Microsoft.AspNetCore.Mvc.ApiExplorer.csproj

```
@@ -7,6 +7,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;aspnetcoremvc</PackageTags>
     <IsPackable>false</IsPackable>
+    <IsAotCompatible>true</IsAotCompatible>
```

```suggestion
    <IsTrimmable>true</IsTrimmable>
```

This is the property we use in the aspnetcore csprojs.

See the top of 

https://github.com/dotnet/aspnetcore/blob/79f745dfd906db54916bf3da2430720eaeda6254/eng/TrimmableProjects.props#L1-L7